applications/MeshingApplication/custom_processes/mmg/mmg_process.h

```diff
@@ -460,26 +427,59 @@ class KRATOS_API(MESHING_APPLICATION) MmgProcess
     }
 
     /**
-     * @brief This method collapses the prisms elements into triangles
+     * @brief This function removes the conditions with duplicated geometries
      */
-    void CollapsePrismsToTriangles();
+    virtual void ClearConditionsDuplicatedGeometries();
 
     /**
-     * @brief This method extrudes the triangles elements into prisms
-     * @param rOldModelPart The old model part
+     * @brief This function creates an before/after remesh output file
+     * @param rOldModelPart The old model part before remesh
      */
-    void ExtrudeTrianglestoPrisms(ModelPart& rOldModelPart);
+    virtual void CreateDebugPrePostRemeshOutput(ModelPart& rOldModelPart);
+
+    ///@}
+    ///@name Protected  Access
+    ///@{
+
+    ///@}
+    ///@name Protected Inquiry
+    ///@{
+
+    ///@}
+    ///@name Protected LifeCycle
+    ///@{
+
+    ///@}
+
+private:
+    ///@name Static Member Variables
+    ///@{
+
+
+    ///@}
+    ///@name Member Variables
+    ///@{
+
+    MmgUtilities<TMMGLibrary> mMmgUtilities;                         /// The MMG utilities class
+
+    ///@}
+    ///@name Private Operators
+    ///@{
+
+    ///@}
+    ///@name Private Operations
+    ///@{
 
     /**
-     * @brief This function removes the conditions with duplicated geometries
+     * @brief This method collapses the prisms elements into triangles
      */
-    void ClearConditionsDuplicatedGeometries();
+    void CollapsePrismsToTriangles();
 
     /**
-     * @brief This function creates an before/after remesh output file
-     * @param rOldModelPart The old model part before remesh
+     * @brief This method extrudes the triangles elements into prisms
+     * @param rOldModelPart The old model part
      */
-    void CreateDebugPrePostRemeshOutput(ModelPart& rOldModelPart);
+    void ExtrudeTrianglestoPrisms(ModelPart& rOldModelPart);
```

Comment:
Git diff looks a bit weird here, there are no functions being remove or replaced in MMG_PROCESS, some of them are moved to "protected" from private.